packages/combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -376,6 +376,10 @@ export const ComboBoxMixin = (subclass) =>
         }
 
         this.__restoreFocusOnClose = true;
+
+        if (this.autoOpenDisabled && !this.filter) {
```

Comment:
Nit: Not immediatly obvious what this does. My guess: User entered something into the input, then opens the combo box, and we need to initialize the focused index then, while when auto open is not disabled, the focused index updates all the time when entering something into the input while the combo box is open? 

On the other hand I see focused index is also updated when auto open is disabled and the combo box is possibly closed (https://github.com/vaadin/web-components/pull/3600/files#diff-bf0d744fa330e331944495a3b4fc16a99bc9f89d618011f89be659cb56b8a734R984), so not sure. Can we clarify this with a comment here?

Reply:
Good point. Your comment made me find a better solution overall. https://github.com/vaadin/web-components/pull/3600/commits/d3e81408a93cacd7f5c54bb2821842566ecbaec2